src/classes/Commands.ts

```diff
@@ -1546,22 +1547,35 @@ export = class Commands {
 
             this.bot.sendMessage(steamID, 'Accepting offer...');
 
+            const partnerId = new SteamID(this.bot.manager.pollData.offerData[offerId].partner);
+            const reply = message.substr(message.toLowerCase().indexOf(offerId) + 11);
```

Comment:
This is super bad since the offer id is always growing. You should instead use `offerId.toString().length` or something similar